docs/USER_GUIDE.md

```diff
@@ -300,6 +301,57 @@ spec:
 - Consider adding authentication and rate limiting for production use
 - Use HTTPS in production environments
 
+### Serving Workflow via HTTP API
+
+The `maestro serve-workflow` command allows you to expose Maestro workflow as HTTP API endpoints, making them accessible via REST API calls. This is useful for integrating workflow into another workflows, web applications, microservices, or other systems that need to communicate with AI workflow.
+
```

Comment:
If we call this `serve-workflow` then for consistency we should call serve `serve-agent`

Maybe `serve` can support both agents and workflow with -an and -w respectively?

Reply:
This is actually just a doc update issue, the command is already unified as `maestro serve` and determines if it's agent or workflow depending on the inputs (@akihikokuroda updated it per @psschwei review above)